kimchi/js/bindings.js

```diff
@@ -2478,8 +2603,15 @@ var caml_pasta_fq_commitments_of_rust = function (x) {
   var w_comm = convertArray(x.w_comm);
   var z_comm = caml_pallas_poly_comm_of_rust(x.z_comm);
   var t_comm = caml_pallas_poly_comm_of_rust(x.t_comm);
+  var caml_lookup;
+  var wasm_lookup = x.lookup;
+  if (wasm_lookup === undefined) {
```

Comment:
`caml_option_of_maybe_undefined` and `caml_option_to_maybe_undefined` could be used.

Reply:
These comments might belong to the code. Wdyt?

Reply:
Changed in https://github.com/o1-labs/snarkyjs-bindings/pull/110/commits/94fc3e027a5d3c70e32dfd22332c1b033a0e0487